src/test/scala/org/kibanaLoadTest/scenario/Dashboard.scala

```diff
@@ -112,10 +112,39 @@ object Dashboard {
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
         )
+        .exec(http("query index pattern meta fields")
+          .get("/api/index_patterns/_fields_for_wildcard")
+          .queryParam("pattern", "kibana_sample_data_ecommerce")
+          .queryParam("meta_fields", "_source")
+          .queryParam("meta_fields", "_id")
+          .queryParam("meta_fields", "_type")
+          .queryParam("meta_fields", "_index")
+          .queryParam("meta_fields", "_score")
+          .headers(headers)
+          .header("Referer", baseUrl + "/app/dashboards")
+          .check(status.is(200))
+        )
+        .exec(http("query index pattern search fields")
+          .get("/api/saved_objects/_find")
+          .queryParam("fields", "title")
+          .queryParam("per_page", "10")
+          .queryParam("search", "kibana_sample_data_ecommerce")
+          .queryParam("search_fields", "title")
+          .queryParam("type", "index-pattern")
+          .headers(headers)
+          .header("Referer", baseUrl + "/app/dashboards")
+          .check(status.is(200))
+        )
```

Comment:
I saw this call only a few times and so far on cloud only, what are the conditions for it to be made?